blog/2023-11-22-config-validation-and-error-handling.md

```diff
@@ -0,0 +1,40 @@
+---
+author: Jan Bouwhuis
+authorURL: https://github.com/jbouwh
+authorImageURL: https://avatars.githubusercontent.com/u/7188918?s=96&v=4
+title: Config processing and error handling
+---
+
+## Changes in component config processing and error handling
+
+The way component YAML configuration is processed has been changed. Now it is possible to raise if an error occurs. Some custom integrations might break if they are using `config.async_process_component_and_handle_errors`.
+Instead they can use `config.async_process_component_and_handle_errors` now. This new method supports raising when an error occurs during config processing.
+From now on failures will no longer be notified as a persistent message, so integrations need to implement error handling to notify users in case of a failure. Notification are still added during setup in case of a config issue.
```

Comment:
```suggestion
The way component YAML configuration is processed has been changed. Now, it is possible to raise if an error occurs. Some custom integrations might break if they are using `config.async_process_component_and_handle_errors`.
Instead, they can use `config.async_process_component_and_handle_errors` now. This new method supports raising when an error occurs during config processing.
From now on, failures will no longer be notified as a persistent message, so integrations need to implement error handling to notify users in case of a failure. Notifications are still added during setup in case of a config issue.
```